BosTau.py

```diff
@@ -0,0 +1,211 @@
+chr26   51992305
+chr27   45612108
+chr28   45940150
+chr29   51098607
+chrX    139009144
+chrMT   16338
+"""
+# Parse list of chromosomes into a list of Chromosome objects which contain the
+# chromosome name, length, mutation rate, and recombination rate
+_chromosomes = []
+
+for line in _chromosome_data.splitlines():
+    name, length = line.split()[:2]
+    _chromosomes.append(stdpopsim.Chromosome(
+        id=name, length=int(length),
+        mutation_rate=1e-8,
+        recombination_rate=1e-8))
+
+# A citation for the chromosome parameters. Additional citations may be needed if
+# the mutation or recombination rates come from other sources. In that case create
+# additional citations with the appropriate reasons specified (see API documentation
+# for stdpopsim.citations)
+
+_assembly_citation = stdpopsim.Citation(
+    doi="FILL ME",
+    year="FILL ME",
+    author="Author et al.",
+    reasons={stdpopsim.CiteReason.ASSEMBLY})
+
+# Create a genome object
+
+_genome = stdpopsim.Genome(
+    chromosomes=_chromosomes,
+    assembly_citations=[_assembly_citation])
+
+# Create a Species Object
+_gen_time_citation = stdpopsim.Citation(
+    doi="",
+    year="",
+    author="",
+    reasons={stdpopsim.CiteReason.GEN_TIME})
+
+_pop_size_citation = stdpopsim.Citation(
+    doi="",
+    year="",
+    author="",
+    reasons={stdpopsim.CiteReason.POP_SIZE})
+
+_species = stdpopsim.Species(
+    id="BosTau",
+    name="Bos Taurus",
+    common_name="B. taurus",
+    genome=_genome,
+    generation_time=5,
+    generation_time_citations=[_gen_time_citation],
+    population_size=62000,
+    population_size_citations=[_pop_size_citation]
+)
+
+stdpopsim.register_species(_species)
+
+
+###########################################################
+#
+# Demographic models
+#
+###########################################################
+
+def _iid():
+    id = "IonaInferredDemography"
+    description = "Iona MacLeod's Inferred Demographic Model for Bos Taurus"
+    long_description = """
+    The Runs of Homozygosity-based infer of Demography from MacLeod et al. 2013.
+    """
+    populations = [
+        stdpopsim.Population(id="FILL ME", description="FILL ME"),
+    ]
+    citations = [
+        stdpopsim.Citation(
+            author="MacLeod et al.",
+            year="2013",
+            doi="https://doi.org/10.1093/molbev/mst125",
+            reasons={stdpopsim.CiteReason.DEM_MODEL})
+    ]
+
+    generation_time = 5
+
+    # parameter value definitions based on published values
+
+    return stdpopsim.DemographicModel(
+        id=id,
+        description=description,
+        long_description=long_description,
+        populations=populations,
+        citations=citations,
+        generation_time=generation_time,
+        population_configurations=[
+            msprime.PopulationConfiguration(
+                initial_size=90, growth_rate=0.0166,
+                metadata=populations[0].asdict())
+        ],
+        #migration_matrix=[
```

Comment:
This can just be `migration_matrix = [0]` if you want to be explicit. Otherwise just delete.